dist/faster_than_csv/PKG-INFO

```diff
@@ -43,4 +43,4 @@ Classifier: Programming Language :: Python :: Implementation :: CPython
 Classifier: Topic :: Software Development
 Requires-Python: >=3.8
 Description-Content-Type: text/markdown
-Version: 21.03.03
+Version: 21.03.04
```

Comment:
This version is the current date `YY.MM.DD`, kinda like the version that PIP uses.

Reply:
OK